index.html

```diff
@@ -121,8 +122,8 @@ <h2>WHAT WE DO</h2>
                   <div class="sub-heading">
                       <p>SOFTWARE</p>
                       <p>We develop your software by using the newest technologies.</p>
-                      <p>MOBILE</p>
-                      <p>Android, iOS or hybrid? We create your app.</p>
+                      <p>INFRASTRUCTURE</p>
+                      <p>Ensure scalability with cloud computing and microservices architecture.</p>

```

Comment:
`resilience and scalability of resource using cloud computing services (AWS, GCP) and microservices architecture.` 
what do you think?